addons/uav/XEH_postInit.sqf

```diff
@@ -0,0 +1,26 @@
+#include "script_component.hpp"
+
+["Air", "init", { // on air vehicle init, add action to class if has pylons
+    params ["_vehicle"];
+    if ((getNumber (configFile >> "CfgVehicles" >> (typeOf _vehicle) >> QUOTE(ADDON) >> "trackPilotCamera")) < 1) exitWith {};
+    private _isUAV = unitIsUAV _vehicle;
+    private _hasPilotCamera = hasPilotCamera _vehicle;
+    if(_isUAV && _hasPilotCamera) then {
+        _pfhID = [{
+            params ["_args", "_pfID"];
+            _args params ["_vehicle"];
+
+            private _pilotCameraTarget = getPilotCameraTarget _vehicle;
+            private _pilotCameraRotation = getPilotCameraRotation _vehicle;
+            if (_pilotCameraTarget select 0) then {
+                private _pilotCameraTargetPos = _pilotCameraTarget select 1;
+                private _pilotCameraTargetRelPos = vectorNormalized ((_vehicle worldToModel (ASLToAGL _pilotCameraTargetPos)) vectorDiff (getPilotCameraPosition _vehicle));
+                _pilotCameraRotation = [ -(rad ((_pilotCameraTargetRelPos select 0) atan2 (_pilotCameraTargetRelPos select 1)) ), -(rad asin(_pilotCameraTargetRelPos select 2))]
+            };
+            _vehicle animateSource ["mainTurret", _pilotCameraRotation select 0];
```

Comment:
Command has global effects, probably don't want to call it all all machines

Reply:
Command is used as it also has the sensors built in, as well as the camera for the PiP on both the UAV dialogue and the 'drone cam' that the operator can have. So if it's used by anyone besides the pilot (potential planned implementation pending), it's necessary that the turret ball be animated as a result.

Reply:
We need some kind of logic to limit the command being run on a single machine
maybe `local _uav` until we have something better

otherwise thing of what will happen with 100 players
10 updates per second * 100 animate sources (each client) * 100 animate destinations (each other client) = 100,000 messages per second

Reply:
Added a local check before the PFH is added, as well as something to terminate the PFH if the vehicle is destroyed.